appraisal/jruby-9.4.0.0.rb

```diff
@@ -0,0 +1,143 @@
+appraise 'rails61-mysql2' do
```

Comment:
Minor: Since we're not actually testing with JRuby 9.4, I would suggest just keeping a `# todo: Actually test JRuby 9.4` in this file for now, as it otherwise seems the stuff here is actually supposed to do something

Reply:
Ah, I meant actually replacing the whole file with a comment; Since we're not using it, this file right now is just a lie :P